tanjun/clients.py

```diff
@@ -561,14 +583,22 @@ def __init__(
         declare_global_commands = declare_global_commands or set_global_commands
         command_ids = command_ids or {}
         if isinstance(declare_global_commands, collections.Sequence):
-            if command_ids and len(command_ids) > 1:
+            if command_ids and len(declare_global_commands) > 1:
                 raise ValueError("Cannot declare global guilds in multiple-guilds and pass command IDs")
```

Comment:
Make this specifically about the declare_global_commands init arg to be less vague